App/PHP_HTML/login sample/forgotPassword.php

```diff
@@ -0,0 +1,161 @@
+            visibility: visible;
+        }
+        .ttip .ttiptext {
+            position: absolute;
+            left: 15px;
+            bottom: -10px;
+        }
+
+        img#tooltip{
+            position: relative;
+            left: -31px;
+            bottom: -1px;
+            height: 27px;
+            width: 27px
+        }
+
+    </style>
+</head>
+<body>
+
+<nav class="navbar navbar-inverse navbar-fixed-top">
+    <div class="container-fluid">
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="../home.php"><img id="img" src="logo3.png"></a>
+        </div>
+        <div id="navbar" class="navbar-collapse collapse">
+            <ul class="nav navbar-nav navbar-right">
+                <li><a href="#">Dashboard</a></li>
+                <li><a href="#">Settings</a></li>
+                <li><a href="#">Profile</a></li>
+                <li><a href="#">Help</a></li>
+            </ul>
+            <form class="navbar-form navbar-right">
+                <input type="text" class="form-control" placeholder="Search...">
+            </form>
+        </div>
+    </div>
+</nav>
+<nav>
+    <div class="content">
+        <form id="form" action="#" method="post">
+            <h2>Forgot Password</h2>
+            <input required type="text" name="input" placeholder="User name or Email" size="47" >
+            <br><br>
+            <input style="display: block; margin:0 auto;" id="sub" type="submit" name="submitform1" value="Submit">
+            <br><br>
+        </form>
+        <form id="form2" action="#" method="post" class="hidden">
+            <h1>Enter New Password for <strong id="name"></strong></h1>
+            <input id="attempt" name="attempt" class="hidden">
+            <input required id="password" type="password" name="user_pass" size="48" pattern="\w{6,}\d+" placeholder="Password">
+            <div class="ttip"><img src="help.png" id="tooltip">
+                <span class="ttiptext">The password must contain at least 1 digit and 6 letters.</span></div>
+            <br><br>
+            <input required  id="confirmPassword" type="password" name="cpassword" size="48" placeholder="Confirm password"><br><br>
+            <input style="display: block; margin:0 auto;" id="sub" type="submit" name="submitform2" value="Submit">
+            <br><br>
+        </form>
+        <?php
+        if($_SERVER['REQUEST_METHOD'] == 'POST') {
+        if(isset($_POST["submitform1"])) {
+        $enteredUserinfo = $_POST["input"];
+        $sql1 = "SELECT * from users where user_name = \"$enteredUserinfo\"";
```

Comment:
This steps automatically lets a user reset the password to any users of this website. Is this a placeholder?